enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/DefaultControllerEnricher.java

```diff
@@ -181,4 +204,20 @@ private String getImagePullPolicy(ResourceConfig xmlResourceConfig, String defau
         }
         return defaultValue;
     }
-}
+
+    private Boolean isAutomaticTriggerEnabled(Boolean defaultValue) {
+        if(((MavenEnricherContext)getContext()).getProperty("fabric8.openshift.enableAutomaticTrigger") != null) {
+            return Boolean.parseBoolean(((MavenEnricherContext) getContext()).getProperty("fabric8.openshift.enableAutomaticTrigger").toString());
+        } else {
+            return defaultValue;
+        }
+    }
+
+    private Long getOpenshiftDeployTimeoutInSeconds(Long defaultValue) {
+        if (((MavenEnricherContext)getContext()).getProperty("fabric8.openshift.deployTimeoutSeconds") != null) {
```

Comment:
Do we really need this cast? Maybe the `getProperty` method should be in base class so we do not need to cast to `MavenEnricherContext`. At least we are not tighting to Maven classes. Same in line 210.